interpreter/cling/lib/Interpreter/DynamicLibraryManager.cpp

```diff
@@ -142,8 +159,26 @@ namespace cling {
     return NPath;
   }
 
-  std::string
-  DynamicLibraryManager::lookupLibrary(llvm::StringRef libStem) const {
+  ///\returns true if substitution was successful.
+  static bool subst(llvm::StringRef value, std::string find,
```

Comment:
Maybe
```suggestion
  static bool substFront(llvm::StringRef value, std::string find,
```
It's a key property of this function that it only replaces text at the *beginning* of the string.

Reply:
Yeah, good point. Fixed.